setup.py

```diff
@@ -289,15 +289,13 @@ def add_dir_name(dir_name, filenames):
 
     # Include directory of module
     mod = SwigPath(module_dir, extension_name)
-    for opts in [swig_opts, extra_compile_args]:
-        opts.extend(["-Isrc/swig/"+extension_name])
 
     libraries = []
     library_dirs = []
     runtime_library_dirs = []
     extra_link_args = []
     define_macros = []
-    extra_include_dirs = ["include"]
+    extra_include_dirs = ["include", "swig"]
```

Comment:
This avoids using `opts.extend(["-Isrc/swig/"+extension_name])` ?

Reply:
indeed, we don't need to add each module directory as an include any longer